api/CSPViolationReportBody.json

```diff
@@ -2,7 +2,7 @@
   "api": {
     "CSPViolationReportBody": {
       "__compat": {
-        "mdn_url": "https://developer.mozilla.org/docs/Web/API/CSPViolationReportBody",
+        "mdn_url": "https://developer.mozilla.org/docs/Web/HTTP/CSP#violation_report_syntax",
```

Comment:
We shouldn't be changing this URL, and instead create a new page that talks about the API itself.

Reply:
OK, can do